ui/plugins/talos/main.html

```diff
@@ -10,7 +10,14 @@
       </ul>
     </li>
   </ul>
-
+  <ul ng-show="perfJobDetail">
+    <li>Perfherder:
+      <ul ng-repeat="line in perfJobDetail | toArray | orderBy: 'suite'">
```

Comment:
You forgot to change 'suite' to 'title' here, when making the corresponding change in controller.js, so the results are no longer sorted.

I'll fix this up for you and push (since we're so close) but try and bear this sort of thing in mind for the future. I can't always tell you *everything* that needs to be done in a review comment, I partly depend on you to test stuff and make sure it all looks good before landing. :)